stardog_fetcher.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+import time
+
+context_str = """kube1
+kube2
+kube3"""
+
+contexts = context_str.split('\n')
+
+for kubecontext in contexts:
+    print("working on {}".format(kubecontext))
```

Comment:
wow so python 2 compliant, so wow

Reply:
oh yeah this is not the cool way to make strings anymore? D:

this didn't exist when Python 2.7 came out, I think